docs/tutorials/Forecast Uber Stock Prices using MindsDB.mdx

```diff
@@ -0,0 +1,89 @@
+# Introduction
+When it comes to AutoML, one of the most remarkable open-source products is MindsDB. MindsDB enhances the existing databases and modifies their tables to function as AI Tables that can train robust Predictor models using the available data, without the need to learn how to code such models in Python, handle their dependencies, or maintain them at a later stage. With MindsDB, you can use simple SQL-like statements and obtain accurate predictions with just a few clicks.
+
+In this tutorial, we will learn how to forecast the stock prices of Uber using MindsDB. The required dataset for this tutorial can be downloaded from here.
+
+# Importing Datasets as Tables in our Database
+
+1. After you've downloaded the Dataset, save the CSV file in a folder Make sure you have MySQL installed on your system, and that it is active. Use MySQL Workbench to test the Database connections and make sure everything is working.
+![image](https://user-images.githubusercontent.com/26898623/236202313-a6378aab-9d10-4bb0-815c-ce859267bc1b.png)
```

Comment:
Instead of using image links, please add all images to /docs/tutorials/images and link them from there.

Reply:
done